From 6750b47d9e0d3074d2e56aa36c476493f533d696 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 15 Jul 2016 16:23:48 -0700
Subject: [PATCH 1/1] scanner: Use uint32_t instead of uint

uint32_t is C99 defined stdint type

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Reviewed-by: Daniel Stone <daniels@collabora.com>
[Rahul Bedarkar: Backported from: 6750b47d9e0d3074d2e56aa36c476493f533d696]
Signed-off-by: Rahul Bedarkar <rahul.bedarkar@imgtec.com>
---
 src/scanner.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/scanner.c b/src/scanner.c
index 6e2c3a3..ebae4cc 100644
--- a/src/scanner.c
+++ b/src/scanner.c
@@ -819,7 +819,7 @@ find_enumeration(struct protocol *protocol,
 	struct interface *i;
 	struct enumeration *e;
 	char *enum_name;
-	uint idx = 0, j;
+	uint32_t idx = 0, j;
 
 	for (j = 0; j + 1 < strlen(enum_attribute); j++) {
 		if (enum_attribute[j] == '.') {
-- 
2.6.2

